step.yml

```
@@ -104,6 +123,62 @@ inputs:
       value_options:
         - "true"
         - "false"
+      is_required: true
+
+  - build_arg:
+    opts:
+      title: Build arguments
+      summary: List of build arguments to be passed to the docker build
```

```suggestion
      summary: List of [build arguments](https://docs.docker.com/build/guide/build-args/) to be passed to the docker build
```
Let's save a Google search for users and add links to the official docs wherever possible